package.json

```diff
@@ -107,12 +108,12 @@
     "rimraf": "5.0.5",
     "rollup": "4.9.4",
     "rollup-plugin-banner2": "1.2.2",
+    "sass": "^1.77.2",
```

Comment:
Package version remains unchanged. Something changed the line it was on. Not sure what it was.